plugins/authenticator/src/u2f_crate/protocol.rs

```diff
@@ -95,7 +95,7 @@ impl U2f {
         challenge: Challenge,
         response: RegisterResponse,
     ) -> Result<Registration> {
-        if expiration(challenge.timestamp) > Duration::seconds(300) {
+        if expiration(challenge.timestamp) > Duration::try_seconds(300).unwrap() {
```

Comment:
```suggestion
        // Safe to unwrap since 300 is within the constraints of Duration::try_seconds
        if expiration(challenge.timestamp) > Duration::try_seconds(300).unwrap() {
```